draft-ietf-scitt-architecture.md

```diff
@@ -930,8 +930,14 @@ Downstream producers benefit from upstream producers providing higher transparen
 
 # Privacy Considerations
 
-Unless advertised by a Transparency Service, every Issuer must treat Signed Statements it registered (rendering them as Transparent Statements) as public.
-In particular, a Signed Statement Envelope and Statement payload MUST NOT carry any private information in plaintext.
+Once a Signed Statement is inserted into the Append-only Log maintained by a Transparency Service, it cannot be removed from the Log.
+The only way to delete the Signed Statement is to delete the entire Log and start a new one.
+
+Therefore, and because usually Transparency Services are publicly accessible, Issuers should treat Signed Statements that they registered (rendering them as Transparent Statements) as public.
```

Comment:
I am not particularly clear about the connect between line 936 start and end of 934, so I recommend avoiding Therefore and because.

Reply:
True. Reordered and clarified.